gulp-tasks/build-packages.js

```diff
@@ -14,59 +14,72 @@ const logHelper = require('./utils/log-helper');
 const pkgPathToName = require('./utils/pkg-path-to-name');
 const rollupHelper = require('./utils/rollup-helper');
 
+const ERROR_NO_BROWSER_BUNDLE = `Could not find the browser bundle: `;
+const ERROR_NO_NAMSPACE = oneLine`
+  You must define a 'browserNamespace' parameter in the 'package.json'.
+  Exmaple: 'workbox-precaching' would have a browserNamespace param of
+  'precaching' in 'package.json'. This will be appended to 'google.workbox'
```

Comment:
`google.workbox` -> `${constants.NAMESPACE_PREFIX}`

(Just in case it changes in the future.)